### PR TITLE
Add license and readme

### LINKED ISSUE #6: Chronagus needs a License
Having no license at all is confusing. I don't know what the license du jour is for RoR projects. But MIT or 3 clause BSD are fine choices. If you want to go full Free Software zealot, GPLv3 is the way to go.

### LINKED ISSUE #14: Add readme
Add a readme with project description and installation instructions. Should also reference the licence (#6)

### PR DESCRIPTION
**Pull request by isilher**
ontsmelten

Closes #6 
Closes #14

### CODE REVIEW

**Comment by isilher** [comment]
@pieterlexis could you have a look at this PR and tell me what you think about the README and LICENSE?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# README
```
Comment (pieterlexis) [inline README.md]: It's better to have the first heading be the name of the project

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+A web based campaign log platform for your roleplaying game.
+- Create campaigns and add players.
+- Create and manage Player Characters, NPC's and Locations.
+- Keep log entries with OC and IC date (for now only based on Pathfinder's [Golarion Calendar](https://pathfinderwiki.com/wiki/Absalom_Reckoning)).
+
+Find a running build on chronagus.plexis.eu, where you can create an account and use all features.
```
Comment (pieterlexis) [inline README.md]: OT: perhaps a more permanent 'testing' environment? One that resets every night at 3:00 CET or something?

Comment (isilher) [reply README.md]: That sounds sweet! I wouldn't know how to do it though.

Comment (isilher) [reply README.md]: I decided to remove the url altogether until the app is more fleshed out.

**Pull request merged**